core/modules/map.rs

```diff
@@ -346,24 +343,25 @@ impl ModuleMap {
           ModuleSource::get_string_source(module_url_found.as_str(), code)
             .map_err(ModuleError::Other)?;
 
-        let (code_cache_info, module_url_found) = if self.enable_code_cache {
```

Comment:
This changed to no longer use `enable_code_cache` when deciding if to fire the `code_cache_ready` callback. I think `enable_code_cache` may still be needed, because on the first run there will be no `code_cache` yet, and we do want the callback to fire.

Reply:
As discussed in DMs, this is now enabled based on whether someone provides a `ModuleSourceCodeCache` or not:

```rs
#[derive(Debug)]
pub struct ModuleSourceCodeCache {
  pub hash: u64,
  pub data: Option<Cow<'static, [u8]>>,
}

// ....
pub struct ModuleSource {
  // ...
  pub code_cache: Option<ModuleSourceCodeCache>,
```

(Probably will rename `ModuleSourceCodeCache` because it's a bad name)